File: go/internal/scoring/testdata/structural-v1/java/DistinctTwo.java
final class DistinctTwo {
  static int run(int value) {
    if (value > 0) {
      if (value % 2 == 0) return value * 2;
    }
    return value;
  }

  static int another(int value) {
    if (value < 0) {
      if (value % 2 != 0) return -value;
    }
    return value;
  }
}

File: go/internal/scoring/testdata/structural-v1/go/distinct_two.go
package calibration

func DistinctTwo(value int) int {
	if value > 0 {
		if value%2 == 0 {
			return value * 2
		}
	}
	return value
}

func AnotherDifficult(value int) int {
	if value < 0 {
		if value%2 != 0 {
			return -value
		}
	}
	return value
}
